wagtail_localize/machine_translators/tests/test_libretranslate_translator.py

```diff
@@ -0,0 +1,93 @@
+from django.test import TestCase, override_settings
+from wagtail.models import Locale
+
+from wagtail_localize.machine_translators import get_machine_translator
+from wagtail_localize.machine_translators.libretranslate import LibreTranslator
+
+
+LIBRETRANSLATE_SETTINGS_ENDPOINT = {
+    "CLASS": "wagtail_localize.machine_translators.libretranslate.LibreTranslator",
+    "OPTIONS": {
+        "LIBRETRANSLATE_URL": "https://libretranslate.org",
+        "API_KEY": "test-api-key",
+    },
+}
+
+
+class TestLibreTranslator(TestCase):
+    @override_settings(
+        WAGTAILLOCALIZE_MACHINE_TRANSLATOR=LIBRETRANSLATE_SETTINGS_ENDPOINT
+    )
+    def setUp(self):
+        self.english_locale = Locale.objects.get()
+        self.french_locale = Locale.objects.create(language_code="fr")
```

Comment:
suggestion: 

```suggestion
        self.french_locale = Locale.objects.create(language_code="fr-fr")
```

so that you can test `self.language_code()`